server/src/index.mjs

```diff
@@ -48,6 +48,8 @@ server.on('upgrade', function upgrade(request, socket, head) {
   } else if ( ! userManagement.isKnownUser(userId) ) {
     logger.warn(`WARNING: Unknown userId: ${userId}; Using default user`);
     userId = config.app.defaultUserId;
+  } else {
+    logger.info(`Using ${userId} user`);
```

Comment:
Probably better to use `Using user ${userId}`

Reply:
done